src/renderers/webxr/WebXRManager.js

```diff
@@ -398,9 +398,15 @@ class WebXRManager extends EventDispatcher {
 				currentPixelRatio = renderer.getPixelRatio();
 				renderer.getSize( currentSize );
 
+				if ( typeof XRWebGLBinding !== 'undefined' ) {
+
+					glBinding = new XRWebGLBinding( session, gl );
```

Comment:
So this change is a preparation for the upcoming WebXR Camera Access support?

How about to bundle everything in one PR instead of filing changes sequentially? This change alone seems not to have any benefit...

Reply:
glBinding is also used by the Depth Sensing module, which I believe is also usable without layers. I'm more than happy to bundle it all in with a more general camera access PR if that's preferred, but I felt I needed to reason this one separately.